### PR TITLE
Initial critical processes folder and components

### PR DESCRIPTION
**Pull request by DapperFox**
Initial folder and very minimal structure for default process. Need help coming up with better structure.

This PR also includes a generic template for creating documents that aren't critical processes. If you want I can do that separately so we can discuss them separately, but they're small enough I think we can have both discussions at the same time.

### CODE REVIEW

**Comment by strikemike2k** [comment]
I choose you, Charizard!

**Review by strikemike2k — APPROVED** [review]
I choose you, Charizard!

File: docs/CriticalProcesses/import.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+A brief overview of the purpose and scope of the documentation.
+
+### Process Overview
+
+A high-level description of the critical software process, including its goals, inputs, outputs, and key players involved.
+
+### Detailed Steps
+
+A step-by-step guide of the process, including all the necessary actions, decision points, and inputs/outputs.
```
Comment (shadoath) [inline docs/CriticalProcesses/import.mdx]:
```suggestion
A step-by-step guide of the process, including all the necessary actions, decision points, and connections.
```

**Review by shadoath — APPROVED** [review]

**Review by walshseanp — APPROVED** [review]
Excellent thank youuuuu

**Pull request merged**